initial-setup.sh

```diff
@@ -2,12 +2,15 @@
 
 ## New computer setup from scratch
 
-# TODO: install Homebrew if doesn't already exist
+# need homebrew to run brew script
+if ! command -v brew >> /dev/null; then
+	/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+fi
 
+# permissions for making `brew.sh` executable in README usage instructions
 ./brew.sh
 
 # `nvm` for easily switching between node versions
 # requires initialization in .bash_profile or .zshrc
-brew install wget
```

Comment:
This is already installed as part of `brew.sh`, so removing duplicate even though will be a no-op